modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_full_width_media_row.yml

```diff
@@ -0,0 +1,56 @@
+id: az_paragraph_full_width_media_row
+label: AZ Paragraph Full Width Media Row
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_full_width_media_row
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_text_media
+
+process:
+  field_az_title:
+    plugin: sub_process
+    source: field_uaqs_summary_short
+    process:
+      delta: delta
+      value: value
+  
+  field_az_media:
+    plugin: sub_process
+    source: field_uaqs_photo
+    process:
+      delta: delta
+      target_id: 
+      - 
+        plugin: migration_lookup
+        source: fid
+        migration: 
+          - az_media
+
+  field_az_text_area/value: 
+    plugin: paragraphs_fw_media_row_field_merge
+    body: field_uaqs_summary
+    link: field_uaqs_links
+
+  field_az_text_area/format:  
+    plugin: default_value
+    default_value: az_standard
+
+  behavior_settings:
+    plugin: paragraphs_behavior_settings
+    bg_text_color: field_uaqs_setting_text_bg_color
+    bg_attach: field_uaqs_setting_bg_attach
+    media_mode: view_mode
```

Comment:
Nitpick: Could we call `media_mode` something like `position`?  Also, I'm not sure whether this needs to be part of the config for the behavior settings process plugin or not since view mode isn't really a field...

Reply:
yes, sure I will change to position. But we need it. because view_mode property mapping is differ for this paragraph type.

Reply:
done